package.json

```diff
@@ -10,17 +10,17 @@
     "author": "Bouvet",
     "license": "UNLICENSED",
     "devDependencies": {
-        "@babel/cli": "7.22.10",
-        "@babel/core": "7.23.6",
-        "@babel/preset-env": "7.22.10",
-        "babel-eslint": "10.1.0",
-        "concurrently": "8.2.1",
+        "@babel/cli": "7.24.8",
+        "@babel/core": "7.25.2",
+        "@babel/preset-env": "7.25.3",
+        "@babel/eslint-parser": "7.25.1",
+        "concurrently": "8.2.2",
         "eslint": "8.56.0",
         "eslint-config-airbnb": "19.0.4",
-        "eslint-plugin-import": "2.28.1",
-        "eslint-plugin-jest": "27.2.3",
-        "eslint-plugin-jsx-a11y": "6.7.1",
-        "eslint-plugin-react": "7.33.2",
+        "eslint-plugin-import": "2.29.1",
```

Comment:
Ser at denne benytter json5@1.0.2, men da regner jeg med den versjonen ikke har den sårbarheten. La også merke til at master bør renames til main, men det er urelatert til denne oppgaven :)

Reply:
Ja litt usikker på hva jeg kan gjøre med at den bruker json5@1.0.2 (?)  Men ja har ikke tilgang til å endre til main selv....

Reply:
I'll rename it to main. I have the power :D

Reply:
Dersom den pakken har sårbarheter kan du override hvilken versjon av json5 eslint-plugin-import bruker: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#overrides, men om den ikke har sårbarheter er jo ikke det nødvendig 😊